Is there a type definition file for @emoji-mart/data in @emoji-mart v5?

I am using emoji-mart v3 in a TypeScript project.
I would like to upgrade to Emoji-mart v5.

```tsx
import data from '@emoji-mart/data'
import Picker from '@emoji-mart/react'

function App() {
  return (
    <Picker data={data} onEmojiSelect={console.log} />
  )
}
```
When I write the above code, I get the following error
Is there any way to upgrade to emoji-mart v5 in a TypeScript project?
<img width="795" alt="スクリーンショット 2022-11-12 19 40 03" src="https://user-images.githubusercontent.com/16137809/201470389-bf9ba559-90be-46f3-9a0e-5e119fa2fd24.png">
> Cannot find module '@emoji-mart/data' or its corresponding type declarations.

> I am using emoji-mart v3 in a TypeScript project. I would like to upgrade to Emoji-mart v5.
> 
> ```tsx
> import data from '@emoji-mart/data'
> import Picker from '@emoji-mart/react'
> 
> function App() {
>   return (
>     <Picker data={data} onEmojiSelect={console.log} />
>   )
> }
> ```
> 
> When I write the above code, I get the following error Is there any way to upgrade to emoji-mart v5 in a TypeScript project? <img alt="スクリーンショット 2022-11-12 19 40 03" width="795" src="https://user-images.githubusercontent.com/16137809/201470389-bf9ba559-90be-46f3-9a0e-5e119fa2fd24.png">
> 
> > Cannot find module '@emoji-mart/data' or its corresponding type declarations.

We were in a similar situation. I still don't know how to get accurate types, but [this solution](https://github.com/missive/emoji-mart/discussions/736) allowed me to at least load the component.

--- a/packages/emoji-mart-data/index.d.ts
+++ b/packages/emoji-mart-data/index.d.ts
@@ -1,0 +1,32 @@
+export interface EmojiMartData {
+  categories: Category[]
+  emojis: { [key: string]: Emoji }
+  aliases: { [key: string]: string }
+  sheet: Sheet
+}
+
+export interface Category {
+  id: string
+  emojis: string[]
+}
+
+export interface Emoji {
+  id: string
+  name: string
+  keywords: string[]
+  skins: Skin[]
+  version: number
+  emoticons?: string[]
+}
+
+export interface Skin {
+  unified: string
+  native: string
+  x?: number
+  y?: number
+}
+
+export interface Sheet {
+  cols: number
+  rows: number
+}

--- a/packages/emoji-mart-data/package.json
+++ b/packages/emoji-mart-data/package.json
@@ -10,6 +10,7 @@
     "directory": "packages/emoji-mart-data"
   },
   "main": "sets/14/native.json",
+  "types": "index.d.ts",
   "scripts": {
     "build": "node ./build.js"
   },

--- a/packages/emoji-mart-react/package.json
+++ b/packages/emoji-mart-react/package.json
@@ -9,7 +9,8 @@
     "url": "https://github.com/missive/emoji-mart",
     "directory": "packages/emoji-mart-react"
   },
-  "source": "react.js",
+  "source": "react.tsx",
+  "types": "dist/index.d.ts",
   "main": "dist/main.js",
   "module": "dist/module.js",
   "scripts": {

--- a/packages/emoji-mart-react/react.tsx
+++ b/packages/emoji-mart-react/react.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import React, { useEffect, useRef } from 'react'
 import { Picker } from 'emoji-mart'
 

